### ISSUE TITLE
nw version 23.5 -load-extension :ERR_BLOCKED_BY_CLIENT

### ISSUE THREAD

**Issue by chatcan** [nwjs/nw.js#6004]
package.json
```
{
  "main": "index.html",
  "name": "canhuang",
  "appname": "canhuang",
  "version": "1.0.0",
  "window": {
    "title": "v1.0.0",
    "toolbar": true,
    "width": 800,
    "height": 800,
    "frame": true
  },
  "chromium-args": "-load-extension=./extensions/ -ignore-certificate-errors",
  "webview": {
    "partitions": [
      {
        "name": "trusted",
        "accessible_resources": [
          "<all_urls>"
        ]
      }
    ]
  }
}
```

index.html

```html
<!DOCTYPE html>
<html lang="zh-CN">

<head>
  <meta charset="utf-8">
</head>

<body>
  <div id="container" class="container"></div>
  <script src="index.js"></script>
</body>

</html>

```

index.js
```javascript
global.contentDocument = document
require('./core.js')

if (nw.App.argv.indexOf('inspect') !== -1) {
  nw.Window.open('about:blank', {
    "show": false,
    "width": 799,
    "height": 799,
  }, (inspectWin) => {
    inspectWin.maximize()
    inspectWin.window.location = "chrome://inspect/#devices"
    inspectWin.show()
  })
}
```

core.js

```javascript
var webview = document.createElement('webview')
var devtools = document.createElement('webview')

webview.setAttribute('style', 'height:300px;width:100%;position:absolute')
devtools.setAttribute('style', 'height:300px;width:100%;position:absolute;top:300px')
devtools.setAttribute('partition', 'trusted')

var container = global.contentDocument.getElementById('container')
container.appendChild(webview)
container.appendChild(devtools)

webview.src = 'https://mp.weixin.qq.com'

const devtoolsviewCommit = () => {
  devtools.removeEventListener('loadcommit', devtoolsviewCommit)
  webview.showDevTools(true, devtools)
}
devtools.addEventListener('loadcommit', devtoolsviewCommit)

devtools.src = 'about:blank'
```

in extension folder create 3 files: devtools.html、manifest.json、custom.html

devtools.html
```html
<html>
<head>
</head>
<body>
<script >
"use strict"

// 添加appdata pannel
chrome.devtools.panels.create("custome",
  "",
  "custome.html",
  function(panel) {
    //console.log(panel)
  }
)

</script>
</body>
</html>
```

manifest.json
```json
{
  "name": "wechat devtools extension",
  "version": "1.1",
  "description": "Extends the Developer Tools, adding appdata",
  "devtools_page": "devtools.html",
  "manifest_version": 2
}

```

custom.html
```html
<html>
<head>
</head>
<body>
custom
</body>
</html>
```

if i use nw version 22.3 to run the project
devtools will add the custom pannel

but when using nw version 23.5 
the custom pannel doesn't appear

use
```
nw23 . inspect
```
to show the inspect window

i see the following error tips

```
chrome-extension://clolcahimhommgjjmgkbkeccoihhkpfg/devtools.html Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
```

is there any mistake ?

**Comment by Christywl** [comment]
I can reproduce this issue on Linux with nwjs-sdk-0.23.5. The custom panel doesn't appear in the devtools  page of the app window.

**Comment by frankhale** [comment]
Related issue: https://github.com/nwjs/nw.js/issues/4878

**Comment by frankhale** [comment]
Here is another simple example to demonstrate the issue: https://github.com/frankhale/nwjs-react-devtools-webview-example

**Comment by rogerwang** [comment]
This is fixed in git and will be available in the next nightly build.

**Comment by frankhale** [comment]
Thank you @rogerwang !!!

**Comment by rogerwang** [comment]
OP's sample is working with this build: https://dl.nwjs.io/live-build/07-24-2017/nw23-890b720-3f64365-bdcc2a7-93e37d9/v0.23.7/

**Comment by rogerwang** [comment]
PS: the "webview" section in package.json should be added into the "manifest.json" file of the extension.

**Comment by rogerwang** [comment]
reopening due to regression.

**Comment by frankhale** [comment]
Thanks @rogerwang!! I just tested react-devtools in 0.27.4 to see if the regression is in there but it's still working as intended.

**Comment by rogerwang** [comment]
This is fixed in git and will be available in the next nightly build.

**Comment by rogerwang** [comment]
This was a regression since 0.25.0 and just fixed in https://dl.nwjs.io/live-build/01-12-2018/nw27-2b7d2db-2f0eedb-a3ef615-547fb57/v0.27.5/

**Comment by rogerwang** [comment]
The last commits fix regression. It will be available in the next nightly build and will be shipped with 0.34.0.
